### ISSUE TITLE
Do not display a number next to a marker, when there is only one marker on the card.

### ISSUE THREAD

**Issue by db0** [kellyelton/OCTGN#113]
In many of my games, I have markers which can only ever be once on the card. Usually these are meant to signify a special status for the card, rather than count sosomething So for example in Call of Cthulhu, I have a "resource" marker, which I assign to cards played as resources, so that I can filter them away from the rest of the cards, when going through all the cards in the table. Having the number next to the marker in that case, and imho in all cases where there is only one marker is unnecessary and less elegant. 

In short, I'd like it so that when there is only one marker on the card, the number "1" is not displayed as well.

*edit by ralig to fix marker typo in subject

**Comment by brine** [comment]
just a recap, this has been reverted as there ended up being more cases where you'd want the number, rather than not wanting the number.  When it gets re-implemented, there will be specific attributes that you give to the markers that dictate whether or not the number 1 should be displayed or not.
